### ISSUE TITLE
Compiling warnings

### ISSUE THREAD

**Issue by groteworld** [HalosGhost/adarcroom#2]
Got it to run but it did have some warnings when compiling

```
In file included from main.c:7:
./main.h:15:8: warning: padding size of 'struct adr_state' with 3 bytes to alignment boundary [-Wpadded]
struct adr_state {
       ^
main.c:40:14: warning: unused variable 'btns' [-Wunused-variable]
    WINDOW * btns [LASER_RIFLE + 1];
             ^
main.c:145:27: warning: unused parameter 'w' [-Wunused-parameter]
update_main_win (WINDOW * w, WINDOW * bs [], enum WIN_TYPE l,
                          ^
main.c:145:39: warning: unused parameter 'bs' [-Wunused-parameter]
update_main_win (WINDOW * w, WINDOW * bs [], enum WIN_TYPE l,
                                      ^
4 warnings generated.
```

![screen shot 2015-05-25 at 1 48 41 am](https://cloud.githubusercontent.com/assets/1421810/7792459/86fd0bd4-0280-11e5-8df3-b816890984b7.png)

Is this as far as you've gotten so far or am I missing something?

A great idea!

**Comment by HalosGhost** [comment]
Yeah, the structure padding warning will likely remain unless I find a couple more bits of state to track.

The other warnings reveal areas where I have yet to finish some work. That is, the variables are unused because I have not yet coded the logic or mechanisms which use them.

At the moment, what you see is as far as I have gotten. Though, once I finally get around to figuring out logic for buttons in ncurses, I can probably get the first stage of the game playable. Unfortunately, development stalled for a bit as I got a job and moved residences. However, now as I have mostly settled again, I should be able (read: get some more motivation) to dive back in.
